pressbooks/pressbooks-network-catalog
# fix: hyphenation for long URLs

Issue #114 

This PR aims to fix an issue with long URLs overflowing the available space in the catalog header

**How to test**

1. Update the catalog page content and add a description with long URL
1. Visit the catalog page and make sure content renders properly without overflowing the space available
1. Make sure book cards content does not overflow as well.
